hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/package-info.java

```diff
@@ -1,6 +1,6 @@
 /**
  * The Hedera implementation of the Hedera Node Schedule Service (as defined in
- * {@link com.hedera.node.app.service.schedule}).
+ * {@code com.hedera.node.app.service.schedule}).
```

Comment:
Links to packages/modules do not work here, probably due to how Gradle calls the Javadoc tool. It's unclear what exactly does not work, but I wasn't able to fix this quickly.

Note: It also does not work on `develop` right now, but there we get a warning that is ignored.